apps/prairielearn/src/pages/administratorJobSequence/administratorJobSequence.ts

```diff
@@ -0,0 +1,28 @@
+import express = require('express');
+import asyncHandler = require('express-async-handler');
+
+import serverJobs = require('../../lib/server-jobs-legacy');
+import { administratorJobSequence } from './administratorJobSequence.html';
+
+const router = express.Router();
+
+router.get(
+  '/:job_sequence_id',
+  asyncHandler(async (req, res) => {
+    const job_sequence_id = req.params.job_sequence_id;
+    const job_sequence = await serverJobs.getJobSequenceWithFormattedOutputAsync(
+      job_sequence_id,
+      null,
+    );
```

Comment:
This is only ever used once, no need for it to be in a separate variable.

```suggestion
    const job_sequence = await serverJobs.getJobSequenceWithFormattedOutputAsync(
      req.params.job_sequence_id,
      null,
    );
```

Reply:
Resolved in commit 6f57e22